src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -13,7 +13,13 @@ type WalletInvoiceValidator = {
 interface IWalletInvoicesRepository {
   persistNew: (invoice: WalletInvoice) => Promise<WalletInvoice | RepositoryError>
 
-  update: (invoice: WalletInvoice) => Promise<WalletInvoice | RepositoryError>
```

Comment:
This change is redundant - field mutability is enforced on the type level https://github.com/GaloyMoney/galoy/blob/main/src/domain/wallet-invoices/index.types.d.ts#L1-L7